### PR TITLE
[ADD] Initial release drafter github actions for automating changelogs

### PR DESCRIPTION
**Pull request by gnomeria**

### CODE REVIEW

File: .github/release-drafter.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+name-template: "v$NEXT_PATCH_VERSION 🌈"
+tag-template: "v$NEXT_PATCH_VERSION"
+categories:
+  - title: "🚀 Features"
+    labels:
```
Comment (cristaloleg) [inline .github/release-drafter.yml]: Does it mean issue label? or PR ?

Comment (gnomeria) [reply .github/release-drafter.yml]: Supposed to be pr labels

Comment (gnomeria) [reply .github/release-drafter.yml]: We could try to edit the label on this pr too if it's to be merged, and see what will it do :)

**Comment by cristaloleg** [comment]
As expected: https://github.com/allegro/bigcache/releases/tag/untagged-c614f34dc86d095ddf49

**Pull request merged**